backend/tournesol/management/commands/sync_yt_playlists.py

```diff
@@ -0,0 +1,93 @@
+import json
+import sys
+
+import google.oauth2.credentials
+import googleapiclient.discovery
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from core.utils.time import time_ago
+from tournesol.models import Entity
+from tournesol.models.entity import TYPE_VIDEO
+
+PLAYLISTS = [
+    {"lang": "en", "playlist_id": "PLXX93NlSmUpayYeaNvKaeftX_QwEei3fU"},
+    {"lang": "fr", "playlist_id": "PLXX93NlSmUpZkR9NaBXMaPlPalESb5tDv"},
+]
+
+
+class Command(BaseCommand):
+    help = "Sync Tournesol recommendations with public playlists on Tournesol Youtube channel"
+
+    def get_videos(self, language):
```

Comment:
Maybe we could use function name which describe a bit more what the function does. (even if a understand what it does from the return 😄)
```suggestion
    def get_top_recent_videos(self, language):
        """Get top recent video that should be present in the YouTube playlist for a certain language."""
```